mocks/mocks.go

```diff
@@ -263,6 +263,21 @@ func (sa *StorageAuthority) GetLatestValidAuthorization(registrationID int64, id
 	return core.Authorization{}, errors.New("no authz")
 }
 
+// GetValidAuthorizations is a mock
+func (sa *StorageAuthority) GetValidAuthorizations(regID int64, names []string, now time.Time) (map[string]core.Authorization, error) {
+	if regID == 1 {
+		auths := make(map[string]core.Authorization)
+		for _, name := range names {
+			if sa.authorizedDomains[name] || name == "not-an-example.com" {
```

Comment:
If you scroll up a bit, it's also special-cased on line 258: https://github.com/letsencrypt/boulder/blob/batched-authz/mocks/mocks.go#L258

The tests fail without that.